AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/PermissionsManager.cs

```diff
@@ -0,0 +1,75 @@
+// The PermissionsManager class implements all the methods needed to 
+// check whether the user has the necessary permissions to access a settingGroup
+// and allows the user to request permission to a settingGroup if already don't have it
+// The user (owner of the project or admin) can then approve or reject the request
+public class PermissionsManager
```

Comment:
```suggestion
public class MySqlPermissionsRepository : IPermissionsRepository
```
I would keep the naming consistent with other repositories and also extend an interface like the other repositories so we're being consistent about design patterns